.github/workflows/auto-build-publish-test.yml

```diff
@@ -13,11 +13,13 @@ concurrency:
 
 jobs:
   Auto-Build-Publish-Test:
+    name: Auto-Build-Publish-Test (${{ matrix.os }}) - (CLI ${{ matrix.cli-version }})
     if: contains(github.event.pull_request.labels.*.name, 'safe to test') || github.event_name == 'push'
     strategy:
       fail-fast: false
       matrix:
         os: [ ubuntu, windows, macos ]
+        cli-version: [ "latest", "2.66.0" ]
```

Comment:
Is 2.66.0 the minimal supported version for this functionality? If not, let's use the minimal version.